docs/release_process.rst

```diff
@@ -0,0 +1,165 @@
+The pull request will be made from the *version_change* branch to the *main* branch.
+The creation of the pull request will trigger a CI build, wait for the CI to finish its checks before merging the pull request.
+If, for some reason, the CI checks fail changes may be required.
+Changes can be made directly to the *vesion_change* branch but quite likely any such changes will need to be propagated to the CI for a permanent fix.
+How changes are propagated to the CI is outside the scope of this document.
+When merging the pull request the *version_change* branch will be automatically deleted.
+
+.. note::
+
+  The merging of a *version_change* pull request created by the CI system is exempt from the 'two reviews' required rule.
+
+When the version number has been set in the *main* branch the preparation of the release can start.
+
+Step 2 - Preparing the release
+==============================
+
+A release is prepared using the *Prepare Release Builder* (:numref:`libcellml_release_process_prepare_release`).
+The *Prepare Release Builder* will create a new branch named *release_staging_<version number>* (where <version number> is an actual semantic version number set in the first step) and generate a changelog.
+
+.. figure:: ./images/release_process/prepare_release_builder.png
+   :align: center
+   :alt: Buildbot prepare release builder interface.
+   :name: libcellml_release_process_prepare_release
+
+   Prepare release builder on Buildbot.
+
+The changelog for a release is generated from information found in merged pull requests between the current release under preparation and the previous release.
+
+.. note::
+
+  The changelog creation is unlikely to be accurate when creating a bug fix/hot fix release on a previously released official version.
+  At this time, that means some manual editing of the changelog will need to happen when creating a bug fix/hot fix release.
+
+.. figure:: ./images/release_process/prepare_release_builder_interface.png
+   :align: center
+   :alt: Buildbot prepare release interface.
+   :name: libcellml_release_process_prepare_release_builder_interface
+
+   Prepare release interface.
+
+There are no options for the prepare release builder, there is only one place you can prepare a release from, there is only one place a release is going to be created.
+The only thing you can do is start a build, (:numref:`libcellml_release_process_prepare_release_builder_interface`).
+
+The prepare release builder will kick off a round of unit tests as part of the preparation process.
+
+When the release has been prepared the *release_staging_<version number>* branch will have been created and a changelog and table of contents entries for the changelog will have been created.
+The changes made for the new changelog should be the only changes from the current *main* branch.
```

Comment:
Should a mention be made to editing the changelog if needed at this point?

Reply:
[0e774c8](https://github.com/cellml/libcellml/pull/1043/commits/0e774c8db5d6c4e376b39ce7ef1500f19d9d32e2)